latest/swarm/google/cloud-deployment.yaml

```diff
@@ -99,5 +99,25 @@ resources:
               workerImage: ranlu/air-tasks:airflow2
               preemptible: True
               sizeLimit: 4
+            - type: synaptor-cpu
+              zone: us-east1-c
+              subnetwork: us-east1
+              machineType: e2-standard-32
+              preemptible: True
+              sizeLimit: 10
+            - type: synaptor-gpu
+              zone: us-east1-c
+              subnetwork: us-east1
+              machineType: n1-highmem-8
+              gpuWorkerAcceleratorType: nvidia-tesla-t4
+              preemptible: True
+              sizeLimit: 10
+            - type: synaptor-seggraph
+              zone: us-east1-c
+              subnetwork: us-east1
+              machineType: n1-highmem-32
+              preemptible: False
+              sizeLimit: 1
+              diskSizeGb: 200
```

Comment:
I can not find where you use the `synaptor-seggraph` cluster in the synaptor dags, how do you use it?

Reply:
For large segmentation runs, I download all of the links between segments in different chunks to one machine for the final connected components step. It isn't necessary for most runs, so I haven't added it to the DAGs yet.

Reply:
All right, I am wondering how are you going to use the disk, if you just start some docker in seuron, you can only mount a temp folder in /tmp which is [tmpfs](https://github.com/seung-lab/seuron_deployment/blob/main/latest/swarm/google/workers.py#L17). So the space is limited by memory not the disk.

Reply:
Ok. I've removed the `diskSizeGb` line from the `-seggraph` node for now and just replaced the instance type with a node that has more memory.